pylib/anki/collection.py

```diff
@@ -1453,13 +1381,7 @@ def flush(self) -> None:
 # legacy name
 _Collection = Collection
 
-
-@dataclass
-class _ReviewsUndo:
-    entries: list[LegacyReviewUndo] = field(default_factory=list)
-
-
-_UndoInfo = Union[_ReviewsUndo, LegacyCheckpoint, None]
+_UndoInfo = Union[LegacyCheckpoint, None]
```

Comment:
Not required for this PR, but I'm wondering whether it might be worth considering dropping support for checkpoints as well in a follow-up. They can cause the user to lose a few minutes of work in a crash, and once they're removed, we can also deprecated the non-undo-supporting methods like card.flush(). WDYT?

Reply:
I think it's reasonable to do that now a lot of legacy code is being removed. I'll look into that in a follow-up PR.